.github/workflows/release.yml

```
@@ -0,0 +1,30 @@
+# name: Build and release (macOS)
```

actionlint (1.7.8)

workflow is empty

(syntax-check)